protocols/token-registry/contracts/TokenRegistry.sol

```diff
@@ -0,0 +1,34 @@
+
+pragma solidity 0.5.10;
+
+import "./interfaces/IAsset.sol";
+
+contract TokenRegistry {
+
+  address owner;
+  event AddToRegistry(bytes4 kind, address contractAddress);
+  event RemoveFromRegistry(bytes4 kind, address contractAddress);
+
+  // Mapping of bytes4 to contract interface type
+  mapping (bytes4 => IAsset) private assetMapping;
+
+  constructor() public {
+    owner = msg.sender;
+  }
+
+  function addToRegistry(bytes4 _kind, IAsset _asset) external {
+    require(msg.sender == owner, "NOT OWNER");
```

Comment:
Suggest `SENDER_MUST_BE_OWNER`

Reply:
Why do we not just use `Ownable`?

Reply:
done

Reply:
I don't see the change - have you pushed?